app/controllers/find/search/locations_controller.rb

```diff
@@ -6,7 +6,9 @@ class LocationsController < Find::ApplicationController
       before_action :providers
       before_action :build_results_filter_query_parameters
 
-      def start; end
+      def start
+        redirect_to find_cycle_has_ended_path if CycleTimetable.find_down?
+      end
```

Comment:
It should be more of 

https://github.com/DFE-Digital/find-teacher-training/blob/e08f05ec6d092abf98e600cc41296c9831d70d6b/app/controllers/application_controller.rb#L9


Keep it down!